neon_homeassistant_skill/locale/skill.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Is this intentionally duplicated from `en-us` or just a leftover?

Reply:
Nope, thought it was deleted ...